src/Data/Component.elm

```diff
@@ -157,23 +160,31 @@ applyTransforms allProcesses transforms (Results materialResults) =
 
                                 -- Note: impacts are always computed from input mass
                                 transformImpacts =
-                                    Impact.sumImpacts
-                                        [ transform.impacts |> Impact.multiplyBy (Mass.inKilograms mass)
-                                        , elec.impacts
-                                            |> Impact.multiplyBy (Energy.inKilowattHours transform.elec)
-                                            |> Impact.multiplyBy (Mass.inKilograms mass)
-                                        , heat.impacts
-                                            |> Impact.multiplyBy (Energy.inKilowattHours transform.heat)
-                                            |> Impact.multiplyBy (Mass.inKilograms mass)
-                                        ]
+                                    [ transform.impacts
+                                    , elec.impacts
+                                        |> Impact.multiplyBy (Energy.inKilowattHours transform.elec)
+                                    , heat.impacts
+                                        |> Impact.multiplyBy (Energy.inMegajoules transform.heat)
```

Comment:
Heat should never be expressed in kWh but in MJ: this is fixed by this patch

Reply:
So this was a bug in the actual code? Is it a data problem that we could avoid/check at the pipeline level?

Reply:
Basically I converted the properly decoded energy expressed in MJ, but expressed it in kWh while the formula expect to keep the same unit. I don't see an easy solution from the data modelization standpoint.